remove_mobile_app_links/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "Remove Odoo mobile app links",
+    "version": "14.0.1.0.0",
+    "category": "Hidden",
+    "summary": "Remove Odoo Enterprise mobile app download links",
+    "website": "https://github.com/OCA/server-brand",
+    "author": "PyTech SRL, Sebastiano Picchi, Alessandro Uffreduzzi",
```

Comment:
The developer names are already included in `CONTRIBUTORS.rst`, so remove them from the `"author"` and only keep the company name.

Reply:
Also when pre-commit will work it will fail because `Odoo Community Association (OCA)`is missing in author value